src/os/ObjectStore.h

```diff
@@ -853,7 +853,7 @@ class ObjectStore {
     /// offset of buffer as aligned to destination within object.
     int get_data_alignment() {
       if (!data.largest_data_len)
-	return -1;
+	return 0;
       return (0 - get_data_offset()) & ~CEPH_PAGE_MASK;
```

Comment:
the next line 857  0 - get_data_offset() is also negative, right?

Reply:
no, it should be ok, because (& ~CEPH_PAGE_MASK) will get a right value